__tests__/spelling-ignore.yml

```diff
@@ -240,6 +244,9 @@
 - 000B
 - 000C
 - 000D
+- 4E00
```

Comment:
For some reason this isn't working. Need to dig into this a bit.

Reply:
I found two different ways to fix this: 

1) In spelling-ignore.yml, replace this:
`- 4E00`
with this:
`- "4E00"`

or: 

2) In spelling.js, replace this: 
`const ignoreConfigured = yaml.safeLoad(fs.readFileSync('./__tests__/spelling-ignore.yml', 'utf8'))`
with this:
`const ignoreConfigured = yaml.safeLoad(fs.readFileSync('./__tests__/spelling-ignore.yml', 'utf8'), 
		{schema: yaml.FAILSAFE_SCHEMA})`

Reply:
I'd much prefer the second which is more generic and will cover future cases, without the risk of somebody wondering why 4E00 is a special kid that needs double quotes. Can be improved with inlined comment in the code to explain that we need this option due to entries starting with a non-zero digit (?)

Reply:
@giacomo-petri It seems if you do the change suggested by Dan, this should pass the test and can go in Call for Review 🎉

Reply:
spelling.js updated with the 2nd option and inline comment added to explain why